src/main/java/org/jabref/gui/filelist/FileListEntry.java

```diff
@@ -48,7 +49,7 @@ public String toString() {
     }
 
     public LinkedFile toParsedFileField() {
-        return new LinkedFile(getDescription(), getLink(), getType().isPresent() ? getType().get().getName() : "");
+        return new LinkedFile(getDescription(), Path.of(getLink()), getType().isPresent() ? getType().get().getName() : "");
```

Comment:
Be careful! This can have unintended consequences, a Linked File "link" can also be an online link!

Reply:
I just noticed that this class isn't used anywhere, so I just delete it. 🤣

Reply:
Probably a leftover from previous refactoring...